octomap_server/src/OctomapServer.cpp

```diff
@@ -64,7 +67,7 @@ OctomapServer::OctomapServer(ros::NodeHandle private_nh_)
   m_occupancyMinZ(-std::numeric_limits<double>::max()),
   m_occupancyMaxZ(std::numeric_limits<double>::max()),
   m_minSizeX(0.0), m_minSizeY(0.0),
-  m_filterSpeckles(false), m_filterGroundPlane(false),
+  m_filterSpeckles(false), m_filterGroundPlane(false), m_simpleGroundFilter(true),
```

Comment:
```suggestion
  m_filterSpeckles(false), m_filterGroundPlane(false), m_simpleGroundFilter(false),
```

I think we should make it act as much like the upstream `octomap_server` by default as possible. We can set this to true when we launch.